src/peft/mapping.py

```diff
@@ -130,6 +130,13 @@ def get_peft_config(config_dict: dict[str, Any]) -> PeftConfig:
 
     return PEFT_TYPE_TO_CONFIG_MAPPING[config_dict["peft_type"]](**config_dict)
 
+def warn_if_tied_embeddings_in_target_modules(model_config, peft_config):
+    if model_config.get("tie_word_embeddings"):
+        for target_module in peft_config.target_modules:
+            if target_module in EMBEDDING_LAYER_NAMES:
+                warnings.warn(
+                    f"{model_config['tie_word_embeddings']=} and a tied {target_module=} is passed to peft config. This can lead to complications, for example when merging the adapter. Are you sure you want to use the target module {target_module}?"
```

Comment:
```suggestion
                    f"{model_config['tie_word_embeddings']=} and the tied {target_module=} is in the peft config. This can lead to complications, for example when merging the adapter. Are you sure you want to use the target module {target_module}?"
```